src/components/Chip/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Pressable from '../Pressable';
+
+export interface IChip extends Omit<React.ButtonHTMLAttributes<HTMLButtonElement>, 'size'> {
+  label: string;
+  size:  'small' | 'large';
+  variant?: 'contained' | 'outlined';
+  color?: 'default';
+  disabled?: boolean;
+  className?: string;
```

Comment:
You can leave props other than `size` as they aren't used in the component. The props can be imported from `Pressable`

Reply:
Done